src/main/resources/travis/travisCentaur.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -e
+set -x
+
+removeCromwellJar() {
+    gsutil rm "${JAR_GCS_PATH}"
+}
+
+# Unpack our credentials and such
+openssl aes-256-cbc -K "$encrypted_fb828d0791bd_key" -iv "$encrypted_fb828d0791bd_iv" -in src/main/resources/travis/jesConf.tar.enc -out jesConf.tar -d
+tar xvf jesConf.tar
+
+# Do a bunch of crap to enable gsutil. It's possible this is overkill but it doesn't take long anyways
+sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 1397BC53640DB551
+CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)"
+echo "deb http://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" | sudo tee /etc/apt/sources.list.d/google-cloud-sdk.list
+curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+sudo apt-get install google-cloud-sdk
+export PYTHONPATH="/usr/lib/python2.7/site-packages:/usr/local/lib/python2.7/site-packages:/usr/local/lib/python2.7/dist-packages"
+export CONFIGURE_OPTS="--enable-unicode=ucs4"
+pyenv install 2.7.10
+pyenv global 2.7.10
+sudo pip install --upgrade pip
+sudo pip install pyopenssl ndg-httpsclient pyasn1 --upgrade
+export CLOUDSDK_PYTHON_SITEPACKAGES=1
+gcloud auth activate-service-account --key-file=broad-dsde-cromwell-dev-d9c443bb4a94.json cromwell@broad-dsde-cromwell-dev.iam.gserviceaccount.com
```

Comment:
If people are leery about our email being public I can embed that in the credentials file as well